odoo-shippable/scripts/build-image.sh

```diff
@@ -144,10 +144,16 @@ git_clone_copy "${PYLINT_REPO}" "master" "conf/pylint_vauxoo_light_vim.cfg" "${R
 git_clone_copy "${PYLINT_REPO}" "master" "conf/.jslintrc" "${REPO_REQUIREMENTS}/linit_hook/travis/cfg/.jslintrc"
 ln -sf ${REPO_REQUIREMENTS}/linit_hook/git/* /usr/share/git-core/templates/hooks/
 
-# Creating virtual environments for python and node js
+# Creating virtual environments for all version installed of python
+# TODO Fix the version 3.2
+for version in '2.7' '3.3' '3.4' '3.5' '3.6'
```

Comment:
3.2?

Reply:
@moylop260 
I try to enable the version "3.2" but show me the follow mistake
![image](https://user-images.githubusercontent.com/1387970/30712758-1517d8a6-9edb-11e7-9a4e-309d1825392c.png)

Installed the package "psutil==4.3.1"
We can in another PR install the python3.2?